scripts/code_generator.rb

```diff
@@ -57,25 +57,27 @@ def append_to_index(root_class_name, index_file)
     root_class_name = ''
     prev_c = nil
     filename = json_path.split('/').last.gsub(/\.json$/, '')
-    filename.split('').each do |c|
-      if prev_c.nil? || prev_c == '.'
-        root_class_name << c.upcase
-      elsif c == '.'
-        # noop
-      else
-        root_class_name << c
+    if !filename.include? "admin.analytics.getFile"
```

Comment:
@seratch let me know if this location for the `if` statement is what you had in mind to skip the `AdminAnalyticsGetFile` generation! I tested the script and it skipped it, but it also didn't add it into the `response/index.ts` file as it was rewriting it - is this expected behavior? also, since I'm importing the interfaces into `packages/web-api/src/response/index.ts` manually to access them in `WebClient.ts` (since `WebClient` cannot import from the `AdminAnalyticsGetFileResponse` file directly), is this ok since this file can be regenerated by running the script and will override those imports?

Reply:
@hello-ashleyintech Good point. The index.ts file needs to be updated correctly. As I mentioned above, you need to add the manually crafted line for AdminAnalyticsGetFileResponse in this script. You can add a new line to do that inside this if statement.